cleanlab/count.py

```diff
@@ -764,8 +769,14 @@ def estimate_confident_joint_and_cv_pred_proba(
                 )
                 missing_class_inds[missing_class] = dup_idx
 
+        # Map validation data into appropriate format to pass into classifier clf
+        if validation_func is None:
+            validation_kwargs = {}
+        elif callable(validation_func):
+            validation_kwargs = validation_func(X_holdout_cv, s_holdout_cv)
+
```

Comment:
normally would not require type-based error checking, but we need to do something here since you're using if/elif structure and there is no "else" statement. 

I'd just add:
```
else:
    raise TypeError("validation_func must be callable function with args: X_val, y_val")
```

Reply:
done!